core/src/main/groovy/noe/server/Tomcat.groovy

```diff
@@ -669,14 +669,18 @@ class Tomcat extends ServerAbstract implements WorkerServer {
     def nl = platform.nl
     def dummyComment = '<!-- Define an AJP 1.3 Connector on port 8009 -->'
     def enableOpenSsl = '<Connector port="' + this.mainHttpsPort.toString() + '" SSLEnabled="true"' + nl +
-        'maxThreads="200" scheme="https" secure="true"' + nl +
-        'SSLCertificateFile="' + this.sslCertificate + '"' + nl +
-        'SSLCertificateKeyFile="' + this.sslKey + '"' + nl +
-        'SSLPassword="' + this.sslKeystorePassword + '"' + nl
+            'maxThreads="200" scheme="https" secure="true"' + nl
+    'maxThreads="200" scheme="https" secure="true"' + nl +
```

Comment:
Both lines are duplicated except for whitespace

Reply:
thanks I hadn't noticed it